src/NUnitFramework/framework/Constraints/ExactCountConstraint.cs

```diff
@@ -43,12 +53,10 @@ public class ExactCountConstraint : PrefixConstraint
         public ExactCountConstraint(int expectedCount, IConstraint itemConstraint)
             : base(itemConstraint)
         {
-            this.expectedCount = expectedCount;
-            this.DescriptionPrefix = expectedCount == 0
-                ? "no item"
-                : expectedCount == 1
-                    ? "exactly one item"
-                    : string.Format("exactly {0} items", expectedCount);
+            Guard.ArgumentNotNull(itemConstraint, "itemConstraint");
```

Comment:
Not to be fixed for this PR, but did we tentatively decide in another project to start using C# 6 for all projects or just that project? If so, we should start using `nameof()` wherever possible.